.github/workflows/validate-pr.yaml

```diff
@@ -0,0 +1,117 @@
+---
+
+name: Validate pull requests
+
+on:
+  pull_request:
+
+jobs:
+  Lint:
+    name: 🧽 Lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        id: cache-dependencies
+        with:
+          python-version: '3.11'
```

Comment:
Is it compatible with 3.12 already? Or is that a whole different beast to tackle.

Reply:
I don't know really, I think updating is a task by itself. There also is a 3.13 already, and we have quite some outdated package versions in the Pipfile. I'll add it to my list of todo's for gh-mpyl